### PR TITLE
Add Python style checking information.

### PR DESCRIPTION
**Pull request by jeffreyoldham**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,11 +71,83 @@ steps in [contributing.md](contributing.md).
...
+
+#### Python
+
+*   Code must be formatted according to the
+    [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html)
+    according to the [yapf formatter](https://github.com/google/yapf).
+*   Code must not generate lint errors or warnings according to
+    [pylint](https://www.pylint.org/) configured for the
+    [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html)
+    as specified in `.pylintrc`.
```
Comment (rsned) [inline README.md:87]: make this a link to the checked in .pylintrc

File: README.md [hunk 2]
```diff
@@ -72,11 +71,83 @@ steps in [contributing.md](contributing.md).
...
+```
+
+To disable yapf for some lines,
+
+```
+# yapf: disable
+... code ...
+# yapf: enable
+```
+
```
Comment (rsned) [inline README.md:145]: Add a TODO to add the equivalent docs for Go code

Comment (jeffreyoldham) [reply README.md:145]: Done.

File: README.md (hunk 1 above)
Comment (jeffreyoldham) [reply README.md:87]: Done.

**Pull request merged**